.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   release:
     # Run the build on macOS, because only the macOS version is built with CGO enabled.
     # See https://github.com/golang/go/issues/12524
```

Comment:
I think this comment is no longer necessary.

Reply:
Think you for review.

Package execution on CI is also changed to be performed by ubuntu runner.
https://github.com/cybozu/assam/pull/263/commits/0611838f8817e778a66cc1b2be6f6885178de538

Reply:
Thanks! I think ci.yaml is good.

However, `release.yml` contains `Run the build on ...` comments are still there. Please check it.

Reply:
Thank you for review.

Delete coment ```Run the build on ...``` on release.yml.
https://github.com/cybozu/assam/pull/263/commits/fe2bc1d55d5763cdeec8ad53b8d8867782aa24d3#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34L10-L12